final class io.netty.bootstrap.FailedChannel$FailedChannelUnsafe extends io.netty.channel.AbstractChannel$AbstractUnsafe
{
final io.netty.bootstrap.FailedChannel this$0;
private void <init>(io.netty.bootstrap.FailedChannel)
{
io.netty.bootstrap.FailedChannel v1;
io.netty.bootstrap.FailedChannel$FailedChannelUnsafe v0;
v0 := @this: io.netty.bootstrap.FailedChannel$FailedChannelUnsafe;
v1 := @parameter0: io.netty.bootstrap.FailedChannel;
v0.<io.netty.bootstrap.FailedChannel$FailedChannelUnsafe: io.netty.bootstrap.FailedChannel this$0> = v1;
specialinvoke v0.<io.netty.channel.AbstractChannel$AbstractUnsafe: void <init>(io.netty.channel.AbstractChannel)>(v1);
return;
}
public void connect(java.net.SocketAddress, java.net.SocketAddress, io.netty.channel.ChannelPromise)
{
java.lang.UnsupportedOperationException v4;
io.netty.bootstrap.FailedChannel$FailedChannelUnsafe v0;
io.netty.channel.ChannelPromise v3;
java.net.SocketAddress v1, v2;
v0 := @this: io.netty.bootstrap.FailedChannel$FailedChannelUnsafe;
v1 := @parameter0: java.net.SocketAddress;
v2 := @parameter1: java.net.SocketAddress;
v3 := @parameter2: io.netty.channel.ChannelPromise;
v4 = new java.lang.UnsupportedOperationException;
specialinvoke v4.<java.lang.UnsupportedOperationException: void <init>()>();
interfaceinvoke v3.<io.netty.channel.ChannelPromise: io.netty.channel.ChannelPromise setFailure(java.lang.Throwable)>(v4);
return;
}
}